### PR TITLE
Remove mentions of nonexistent test

### PR DESCRIPTION
**Pull request by mbzomowski**
# Description

Removed any mention of the `tf.nightly-resnet-imagenet-func-v2-8` test, which has been removed.

# Tests

# Checklist

Before submitting this PR, please make sure (put X in square brackets):
- [X] I have performed a self-review of my code.
- [X] I have necessary comments in my code, particularly in hard-to-understand areas.
- [] I have run one-shot tests and provided workload links above if applicable. 
- [X] I have made or will make corresponding changes to the doc if needed.

### CODE REVIEW

**Comment by gkroiz** [comment]
@mbzomowski could you also attach a log for this oneshot test using the shortcuts instruction when creating the PR

File: tests/README.md [hunk 1]
```diff
@@ -38,7 +38,7 @@ correctly, or to extract the correct name to run a one shot test.
 To manually run one shot tests, first [connect to a cluster](https://console.cloud.google.com/kubernetes/list) and then run the following:
 
 ```bash
-export TEST_NAME=tf.nightly-resnet-imagenet-func-v2-8
+export TEST_NAME=tf.nightly-vit-beans-func-v4-8-1vm
 jsonnet tests/oneshot.jsonnet -J . -S --tla-str test=$TEST_NAME | kubectl create -f -
 ```
```
Comment (gkroiz) [inline tests/README.md:44]: I think we should also adjust the job id to match with the test_name used in the one shot tutorial

Comment (mbzomowski) [reply tests/README.md:44]: Done

File: tests/README.md [hunk 2]
```diff
@@ -38,7 +38,7 @@ correctly, or to extract the correct name to run a one shot test.
 To manually run one shot tests, first [connect to a cluster](https://console.cloud.google.com/kubernetes/list) and then run the following:
 
 ```bash
-export TEST_NAME=tf.nightly-resnet-imagenet-func-v2-8
+export TEST_NAME=tf.nightly-vit-beans-func-v4-8-1vm
```
[the remark this answers is not in the document]
Comment (RissyRan) [reply tests/README.md]: I don't think so. Ideally we should provide a real example for customers. But due to the special case discussed offline, it should be fine.

File: tests/README.md [hunk 3]
```diff
@@ -38,17 +38,17 @@ correctly, or to extract the correct name to run a one shot test.
 To manually run one shot tests, first [connect to a cluster](https://console.cloud.google.com/kubernetes/list) and then run the following:
 
 ```bash
-export TEST_NAME=tf.nightly-vit-beans-func-v4-8-1vm
+export TEST_NAME=tf.nightly-dlrm-criteo-conv-v100-x1
 jsonnet tests/oneshot.jsonnet -J . -S --tla-str test=$TEST_NAME | kubectl create -f -
 ```
 
-The above command will generate a job id such as `job.batch/pt-nightly-unet3d-conv-v3-8-1vm-gz8ww`. To find the detail of the test, search in [GoogleCloud->Kubernetes->workload in the project `xl-ml-test`](https://console.cloud.google.com/kubernetes/workload/overview?mods=allow_workbench_image_override&project=xl-ml-test) with the job id `pt-nightly-unet3d-conv-v3-8-1vm-gz8ww`.
+The above command will generate a job id such as `job.batch/tf.nightly-dlrm-criteo-conv-v100-x1`. To find the detail of the test, search in [GoogleCloud->Kubernetes->workload in the project `xl-ml-test`](https://console.cloud.google.com/kubernetes/workload/overview?mods=allow_workbench_image_override&project=xl-ml-test) with the job id `tf.nightly-dlrm-criteo-conv-v100-x1`.
```
Comment (gkroiz) [inline tests/README.md:45]: small nit: in the current example we see `job.batch/pt-nightly-unet3d-conv-v3-8-1vm-gz8ww` (namely the gz8ww, which isn't part of the test name). The `gz8ww` is there to uniquely identify different one-shots. I suggest we leave that in. While technically there is not an actual one-shot with the name `job.batch/tf.nightly-dlrm-criteo-conv-v100-x1-gz8ww`, it is fine as an example of what a job id might look like.

Comment (mbzomowski) [reply tests/README.md]: Whoops, thought I left that in

**Review by gkroiz — APPROVED** [review]

**Review by RissyRan — APPROVED** [review]

**Pull request merged**